static/docs/use-cases/using-shared-server-for-the-team.md

```diff
@@ -0,0 +1,94 @@
+# Using Shared Server For The Team
```

Comment:
It took me a while to recall what's this about. Shared Server is not very obvious name for this scenario. Let's use a straightforward name, something like: `Multiple Data Scientists on a Single Machine`

Reply:
Ok